src/pygui.jl

```diff
@@ -130,7 +132,9 @@ end
 
 # Qt: (PyQt5, PyQt4, or PySide module)
 function qt_eventloop(QtCore::Py, sec::Real=50e-3)
-    fixqtpath()
+    # `fixqtpath()` seems to not be working,
+    # https://github.com/JuliaPy/PythonPlot.jl/issues/17
+    #fixqtpath()
```

Comment:
It is it not working, or not necessary?  What is the downside of keeping this function call (which may still be needed on some systems)?

Reply:
It somehow breaks the Qt5 backend for me, and apparently others as described in #17 . It isn't necessary for me; I don't even need to set `QT_PLUGIN_PATH`. After spending a few hours debugging the first problem, I didn't take time to investigate what is going here with qt path.

Reply:
It somehow breaks the Qt5 backend for me, and apparently others as described in #17 . It isn't necessary for me; I don't even need to set `QT_PLUGIN_PATH`. After spending a few hours debugging the first problem, I didn't take time to investigate what is going here with qt path.